updater/reports/ReportOrgHistory.py

```diff
@@ -0,0 +1,55 @@
+from .ReportDaily import *
+
+# Lists the total number of organizations
+class ReportOrgHistory(ReportDaily):
+	def name(self):
+		return "org-history"
+
+	def updateDailyData(self):
+		newHeader, newData = self.parseData(
+			self.executeQuery(self.query())
+		)
+		self.header = newHeader if newHeader else self.header
```

Comment:
`self.header` is not previously defined. Therefore, we would not set anything in the else part, right?